src/main/java/org/apache/commons/io/input/BrokenInputStream.java

```diff
@@ -136,9 +136,9 @@ private RuntimeException rethrow() {
     /**
      * Throws the configured exception.
      *
-     * @param n ignored
-     * @return nothing
-     * @throws IOException always thrown
+     * @param n ignored.
+     * @return nothing.
+     * @throws IOException as configured.
```

Comment:
This is confusing compared to the previous Javadoc IMO because it is in fact the case that _an_ exception is _always_ thrown. You could say instead "Always throws the exception supplied from a constructor."

Reply:
Done